devlab/tests/testcases/test_resource_migration.py

```diff
@@ -372,7 +372,7 @@ def test_glance_images_not_in_filter_did_not_migrate(self):
                             'in filter, Image info: \n{}'.format(image))
 
     def test_migrate_neutron_networks(self):
-        """Validate networks were migrated with correct parameters.
+        r"""Validate networks were migrated with correct parameters.
```

Comment:
There are two ways to resolve this pylint issue - will try another one in next iteration